### PR TITLE
Typography tweaks

### PR DESCRIPTION
**Pull request by joecritch**
Hey! Just a quick effort of improving the typography slightly.

- Changed font from "Open Sans" to "Source Sans Pro". Feels neater and friendlier, coupled with some subtle tracking/leading changes.
- Fixed font loading on non-Download pages by moving the font loader to a global JS file.
- Reduced the size of the header logo
- Dimmed the hover/focus nav item background color so it doesn't match the body background

**Before:**
before

**After:**
after

### CODE REVIEW

**Comment by patrickkettner** [comment]
awesome!

I actually prefer the larger logo personally - @ryanseddon @SlexAxton @stucox opinions?

What does the small screen view look like with the logo?

**Comment by SlexAxton** [comment]
I like the smaller logo because of the alignment with the rest of the menu.
If it could be bigger to appease Kettner, and keep that horizontal
alignment I'd be fine with that too.

**Comment by patrickkettner** [comment]
haha, im fine with the smaller logo if it is 2 vs 1 :]

**Comment by ryanseddon** [comment]
I like the smaller logo too, sorry @patrickkettner :p 

Also is there anything we can do about the abysmal contrast ratio? http://leaverou.github.io/contrast-ratio/#%23AAA8A9-on-%23ECECEC

**Comment by joecritch** [comment]
Sure, a couple more updates:
- Increased contrast ratio of the hero sub-heading
- Horizontally aligned the logo (and reduced its container width to match)

(I'm PR'ing master into master here. I'm guessing that's bad?)

**Comment by patrickkettner** [comment]
> (I'm PR'ing master into master here. I'm guessing that's bad?)

nah, not when its two different repos.

File: frontend/styl/header.styl [hunk 1]
```diff
@@ -44,7 +51,7 @@
   &:hover,
   &.active {
     outline: none
-    background: $BackgroundGrey
+    background: #f7f7f7
```
Comment (ryanseddon) [inline frontend/styl/header.styl]: Can you add that colour to `vars.styl`

File: frontend/styl/header.styl [hunk 2]
```diff
@@ -85,7 +97,7 @@
     flex: 0
     min-width: 155px
     a:focus .logo {
-      background-position: right 40% center
+      // background-position: right 40% center
```
Comment (ryanseddon) [inline frontend/styl/header.styl]: Can you remove this lock if we don't need that anymore

File: frontend/styl/home.styl [hunk 3]
```diff
@@ -31,11 +32,11 @@
 }
 
 .home-heading {
-  max-width: 15em
+  max-width: 12em
 }
 
 .home-subheading {
-  color: $Grey
+  color: #666
```
Comment (ryanseddon) [inline frontend/styl/home.styl]: Into `vars.styl` please

File: frontend/styl/home.styl [hunk 4]
```diff
@@ -55,6 +59,11 @@
 
   p {
     line-height: 1.5
+    color: #666
```
Comment (ryanseddon) [inline frontend/styl/home.styl]: `vars.styl`

File: frontend/templates/partials/head.hbs [hunk 5]
```diff
@@ -4,5 +4,5 @@
 {{! `this` refers to the arg passed to the helper - in this case just a string}}
 <title>{{this}}</title>
 <link  rel="stylesheet" href="/css/main.css" />
-<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+<link href='http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,600&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
```
Comment (ryanseddon) [inline frontend/templates/partials/head.hbs]: Can you  make this https

**Comment by joecritch** [comment]
Cheers @ryanseddon, cleanup done.

**Comment by ryanseddon** [comment]
Nice :+1:

File: frontend/templates/index.hbs [hunk 6]
```diff
@@ -28,5 +28,6 @@
       </div>
     </main>
     {{> partials/footer}}
+    <script src="/js/build.js"></script>
```
Comment (ryanseddon) [inline frontend/templates/index.hbs]: This would make sense to be in the `footer` partial since it's required on every page.

**Comment by patrickkettner** [comment]
@joecritch wanna add that to the footer?

**Comment by joecritch** [comment]
@patrickkettner @ryanseddon Done, cheers.

**Pull request merged**